tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "esModuleInterop": true,
-    "jsx": "preserve",
+    "jsx": "react",
```

Comment:
will this cause an issue? we preseve so that the bundler in the target application can do the transform

Reply:
Good point. I changed it for the sake of my demo, but I guess I should really override it there somehow 🤔

Reply:
Sorry not the demo, it was for jest. I should be able to configure jest to do the transform instead